# Goal
# Set up enums
uint8 ARM=1
uint8 DISARM=0

# Arm/Disarm
uint8 arm_cmd

---
# Result
# Success/Fail
uint8 ARMED=1
uint8 DISARMED=0

uint8 final_state

---
# Feedback
uint8 curr_status
